src/lib/utilities.js

```diff
@@ -63,7 +64,15 @@ export const getContents = (content, contents) => {
 };
 
 export const getQuestions = questions =>
-  questions ? [{ questions, layout: { row: 'middle', col: 'left' } }] : null;
+  questions
```

Comment:
instead of providing a `questions` object that contains an array of questions and a single layout, each question now has a layout embedded into the question object and the `questions` is an array of `question` objects which is in line with how other page content is managed.

this is because in the JSON questions are represented already by an array and so to add a top-level layout to all of the questions would require re-writing ~150 JSON files. An added benefit is that we can add a layout to individual questions instead of positioning just the entire block of questions.